# HG changeset patch
# User Rik <rik@octave.org>
# Date 1400185715 25200
#      Thu May 15 13:28:35 2014 -0700
# Branch stable
# Node ID a6d13c62fccf1ef24eeeac515ac58c94055738d0
# Parent  83c85d95ac7bae52c00433a63b7b8dd8270c8474
Update CITATION file to include Rik Wehbring, use current version number.

* CITATION: Update CITATION file to include Rik Wehbring, use current version
number.

* etc/CHECKLIST: Update checklist for a new release to include updating the
CITATION file.

diff --git a/CITATION b/CITATION
--- a/CITATION
+++ b/CITATION
@@ -1,20 +1,21 @@
 To cite GNU Octave in publications use:
 
-  John W. Eaton, David Bateman, and Søren Hauberg (2009).  GNU Octave version
-  3.0.1 manual: a high-level interactive language for numerical computations.
+  John W. Eaton, David Bateman, Søren Hauberg, Rik Wehbring (2014).
+  GNU Octave version 3.8.1 manual: a high-level interactive language for
+  numerical computations.
   CreateSpace Independent Publishing Platform.  ISBN 1441413006,
   URL http://www.gnu.org/software/octave/doc/interpreter/
 
 A BibTeX entry for LaTeX users is:
 
   @book{,
-    author    = {John W. Eaton and David Bateman and S\oren Hauberg},
-    title     = {{GNU Octave} version 3.0.1 manual: a high-level interactive language for numerical computations},
+    author    = {John W. Eaton, David Bateman, S\oren Hauberg, and Rik Wehbring},
+    title     = {{GNU Octave} version 3.8.1 manual: a high-level interactive language for numerical computations},
     publisher = {CreateSpace Independent Publishing Platform},
-    year      = {2009},
+    year      = {2014},
     note      = {{ISBN} 1441413006},
     url       = {http://www.gnu.org/software/octave/doc/interpreter},
   }
 
 We have invested a lot of time and effort in creating GNU Octave, please cite it
 when using it.  See also `citation pkgname' for citing Octave packages.
diff --git a/etc/CHECKLIST b/etc/CHECKLIST
--- a/etc/CHECKLIST
+++ b/etc/CHECKLIST
@@ -1,12 +1,14 @@
 Things to do before making a release:
 
   * Update the version number and release date in configure.ac.
 
+  * Update version number and release year in CITATION.
+
   * Update the NEWS file.
 
   * Update the NEWS-<VERSION>.html, index.in, news.in, and download.in
     files for the web site.
 
   * Copy tarballs to ftp.gnu.org.
 
   * Post a release announcement to the help@octave.org and
